CHANGELOG.rst

```diff
@@ -35,6 +35,7 @@ Added
 - Improve settings documentation styling (#251)
 - Add a ihatemoney delete-project command to delete a project (#375)
 - Add nice 404 error pages (#379)
+- Document database migrations (#390)
```

Comment:
Please rebase and make sure it is in the new version section of the changelog :-)